plugins/source/gcp/resources/plugin/tables.go

```diff
@@ -128,6 +128,7 @@ func PluginAutoGeneratedTables() []*schema.Table {
 		containeranalysis.Occurrences(),
 		dns.Policies(),
 		dns.ManagedZones(),
+		dns.ResourceRecordSets(),
```

Comment:
I'm not 100% sure if I should add this here also, or if the relationship with the managed zones enought. Can you confirm, please?

Reply:
The relationship with managed zones is enough, it shouldn't be added here as well (otherwise I would expect the sync to fail because there is no associated parent for the resolver to use)